src/main/groovy/com/palantir/gradle/javadist/DistributionExtension.groovy

```diff
@@ -25,6 +25,7 @@ class DistributionExtension {
     private List<String> defaultJvmOpts = []
     private boolean enableManifestClasspath = false
     private String javaHome = null
+    private List<String> excludeFromVar = ["log", "run"]
```

Comment:
Groovy Strings -> Strings. Also, I think we should consider *always* not packaging these, so probably want to do the list adjustment here or leave existing excludes.